testing/cfitsio/APKBUILD

```diff
@@ -0,0 +1,55 @@
+# Contributor: Holger Jaekel <holger.jaekel@gmx.de>
+# Maintainer: Holger Jaekel <holger.jaekel@gmx.de>
+pkgname=cfitsio
+pkgver=3.47
+pkgrel=0
+pkgdesc="A library of C and Fortran subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format"
+url="https://heasarc.gsfc.nasa.gov/fitsio/"
+arch="all"
+license="custom"
+makedepends="
+	bzip2-dev
+	curl-dev
+	gfortran
+	"
+subpackages="
+	$pkgname-static
+	$pkgname-dev
+	"
+source="https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/$pkgname-$pkgver.tar.gz"
+
+build() {
+	./configure \
+		--prefix=/usr \
+		--with-bzip2 \
+		--enable-reentrant \
+		--enable-sse2 \
+		--enable-ssse3 \
+		--enable-hera
+	make shared
+	make utils
+}
+
+package() {
+	make DESTDIR="$pkgdir" install
+	mkdir -p "$pkgdir"/usr/share/licenses/cfitsio/
+	cp License.txt "$pkgdir"/usr/share/licenses/cfitsio/
+	rm "$pkgdir"/usr/bin/testprog
+	rm "$pkgdir"/usr/bin/cookbook
+
+	# avoid conflicts with smem
+	rm "$pkgdir"/usr/bin/smem
+}
+
+static() {
+	pkgdesc="$pkgdesc (static library)"
+	mkdir -p "$subpkgdir"/usr/lib
+	mv "$pkgdir"/usr/lib/*.a "$subpkgdir"/usr/lib
+}
```

Comment:
That should be automatically generated if you add `$pkgname-static` to `subpackages`

Reply:
function deleted